homeassistant/components/binary_sensor/nest.py

```diff
@@ -128,6 +128,11 @@ def device_class(self):
         """Return the device class of the binary sensor."""
         return _VALID_BINARY_SENSOR_TYPES.get(self.variable)
 
+    @property
+    def unique_id(self):
```

Comment:
This property should move to `NestSensorDevice` at
https://github.com/home-assistant/home-assistant/blob/b91a061cef55d38fa70a49b749c200eb793bf068/homeassistant/components/nest/__init__.py#L275